input/docs/handbook/view-models/boilerplate-code/index.md

```diff
@@ -0,0 +1,59 @@
+If you are tired of writing boilerplate code for property change notifications, you can try either <a href="https://github.com/Fody/PropertyChanged">PropertyChanged.Fody</a> or <a href="https://www.nuget.org/packages/ReactiveUI.Fody/">ReactiveUI.Fody</a>. These libraries are both based on <a href="https://github.com/Fody/">Fody</a> - an extensible tool for weaving .NET assemblies, and they'll inject `INotifyPropertyChanged` code into properties at compile time for you. We recommend using <a href="https://www.nuget.org/packages/ReactiveUI.Fody/">ReactiveUI.Fody</a> package that also handles `ObservaleAsProperyHelper` properties.
```

Comment:
Awesome, one suggestion is maybe get rid of the input/docs/view-models/boilerplate-code/index.md and just do

input/docs/view-models/boilerplate-code.md

it should be indexed as normal.

Reply:
The contents of the text looks good.

Reply:
I didn't know that could work! Changing that right now!